framework/wazuh/core/cluster/master.py

```diff
@@ -737,10 +743,24 @@ async def sync_integrity(self, task_id: str, received_file: asyncio.Event):
                                       cls=c_common.WazuhJSONEncoder).encode()
                 await self.send_request(command=b'syn_m_c_r', data=task_id + b' ' + exc_info)
             finally:
+                try:
+                    self.interrupted_tasks.remove(task_id)
```

Comment:
```suggestion
                    # In case task was interrupted, remove its ID from the interrupted set.
                    self.interrupted_tasks.remove(task_id)
```